Automattic/jetpack
# Instant Search: Fix search on private simple sites

Fixes #17269.

#### Changes proposed in this Pull Request:
* Temporarily forks `wpcom-proxy-request` to fix a bug caused by two instances of `wpcom-proxy-request` running on the same window. Also, strips unnecessary libraries from the request package.
* Moves `query-string-decode` into an externals folder.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Follow the testing instructions in D50535-code.
* Ensure that search works as expected on your Jetpack site.

#### Proposed changelog entry for your changes:
* None.
